backend/Repositories/SightingRepo.cs

```diff
@@ -24,5 +27,25 @@ public IEnumerable<Sighting> GetAllSightings()
                 .Sightings
                 .Include(s => s.Species);
         }
+
+         public Sighting GetById(int id)
+        {
+            return _context
+                .Sightings
+                .Single(s => s.Id == id);
+        }
```

Comment:
Actually, why did you create this method and then not use it? If you're not using it, please delete it!